.changes/unreleased/ENHANCEMENTS-20250110-134832.yaml

```diff
@@ -0,0 +1,5 @@
+kind: ENHANCEMENTS
```

Comment:
I think write-only attributes hasn't technically been launched yet: https://github.com/hashicorp/terraform/blob/main/.changes/unreleased/NEW%20FEATURES-20250107-125354.yaml.

So, I don't think we need a separate CHANGELOG entry for this. If it's worth calling out, I'd add it to the existing entry.